README.md

```diff
@@ -1689,6 +1689,7 @@ _Related: [Static site generators](#static-site-generators)_
 - [fx](https://github.com/metrue/fx) - fx is a tool to help you do Function as a Service with painless on your own servers. `MIT` `Go`
 - [IronFunctions](https://github.com/iron-io/functions) - The serverless microservices platform by [iron.io](https://www.iron.io/). `Apache-2.0` `Go`
 - [LocalStack](https://localstack.cloud/) - LocalStack is a fully functional local AWS cloud stack. This includes Lambda for serverless computation. ([Source Code](https://github.com/localstack/localstack)) `Apache-2.0` `Python/Other`
+- [Nhost](https://nhost.io/) - The Open Source Firebase Alternative with GraphQL. ([Source Code](https://github.com/nhost/nhost)) `MIT` `Docker/Nodejs/Go`
```

Comment:
The description should explain what the software does.

Reply:
Added changes.